ChangeLog.md

```diff
@@ -1,8 +1,145 @@
 # Changelog for haskell-language-server
 
+## 2.1.0.0
+
+* Binaries for GHC 9.4.6
+* Completions for .cabal files
+* Performance improvements
+
```

Comment:
```suggestion
* Show package name and its version while hovering on import statements ([#3691](https://github.com/haskell/haskell-language-server/pull/3691))

```

It would be great to highlight this because it's a new UI-perceivable feature.